src/app/communities/communities.component.html

```diff
@@ -0,0 +1,20 @@
+<md-card *ngFor="let community of communities">
+  <md-card-header>
+    <img md-card-avatar src="{{ community.logo }}">
+    <md-card-title>
+      {{ community.name }}
+      <md-icon *ngIf="community.codeOfConduct">security</md-icon>
+    </md-card-title>
+    <md-card-subtitle>Beginner, Intermediate</md-card-subtitle>
```

Comment:
I wouldn't hard code this. I'm sure as the list grows there will be some groups that are only for beginners or only for intermediates, etc. I'd grab this from the `communities.json`